Create a change log :)

A-Zak opened the pull request.

A-Zak commented:
Fair enough, conventions are important :)

tibetsprague commented on CHANGELOG.md:
```diff
@@ -0,0 +1,9 @@
+  - Misc
+
+### Unreleased
+  - Featrures Added
```
mispelled

tibetsprague commented on CHANGELOG.md:
```diff
@@ -0,0 +1,9 @@
+### template
+  - Fetures Added
```
mispelled

tibetsprague commented on CHANGELOG.md:
```diff
@@ -0,0 +1,9 @@
+### template
+  - Fetures Added
+  - Bugs Fixed
+  - Maintenence / Refactor
```
mispelled

tibetsprague commented on CHANGELOG.md:
```diff
@@ -0,0 +1,9 @@
+### template
```
does this mean, for each version show the following list? i think it would be better to just start the changelog with the first release

A-Zak commented:
@tibetsprague 
Fixed the spelling errors 😅

About the structure - I'm not entirely sure what you meant in your comment, so let me try to explain what I had in mind and you tell me if it makes sense :)

I figured that eventually, we'd have this structure: 
```
#templeate 
  <the list of categories we agree should be there>
#unreleased 
  <The things in dev that haven't yet been released, split by the categories above>
#verion 0.2.3
  <changes in this version
#verion 0.2.2
  <changes in this version
#verion 0.2.1
  <changes in this version
```

So the unreleased section is what we move down to its own version when we deploy it...
The template is there just to make it easy to recreate the unreleased section (just copy-paste the template instead of typing it out manually)...


What do you think / does this make sense?

A-Zak commented:
@jellegerbrandy github's release system requires for someone to input the content of a release once it is released, which brings us right back to the question this change log wants to answer - "what was changed in this release?"...

I'm actually happy with the "just a list of stuff that changed in the past"... this is exactly what this changelog is here to answer...

And about the categorisation, what work are you talking about? The mental work of figuring out which category to put what you're working on? 🤣 
Splitting these changes into two categories helps understand in which direction we're spending our time working on... And helps users digest information better / focus on what's more important to them...

p.s.
I've actually started with 6 categories in mind and boiled it down 2 categories to make it simple...
